draft-ietf-masque-connect-udp-listen.md

```diff
@@ -164,6 +166,20 @@ and therefore is to be ignored). This document does not define any parameters
 for the connect-udp-listen header field value, but future documents might define
 parameters. Receivers MUST ignore unknown parameters.
 
+
+# The proxy-public-address Response Header Field {#response}
+
+Upon accepting the request, the proxy MUST allocate either one or both of IPv4address to port and IPv6address to port pairings. IPv4address, IPv6address and port are defined in {{!URI=RFC3986}}
+
+The proxy must then respond with the allocated IP and port pairings using the "proxy-public-address" response header defined as follows:
+
+~~~ ascii-art
+proxy-public-address = IP-port-pairing (, IP-port-pairing)
```

Comment:
This encoding allows sending two addresses of the same address family, which doesn't really make sense. That might be a pain to encode in the ABNF format but we can fix that with a MUST